app/code/Magento/StoreGraphQl/Model/Resolver/StoreConfigResolver.php

```diff
@@ -52,7 +61,10 @@ public function resolve(
         array $args = null
     ) : Value {
 
-        $storeConfigData = $this->storeConfigDataProvider->getStoreConfig();
+        $storeConfigData = array_merge(
```

Comment:
We should use just one config provider and the list of accessible fields should be filtered according to the extensible di.xml

Reply:
Hi @paliarush 
God point
The decision with one config provider doesn't feet here. 
We getting configs from different sources, StoreConfigDataProvider gets data from StoreConfigInterface unlike ExtendedStoreConfigDataProvider which gets data from ScopeConfigInterface. 
It is not possible to get all StoreConfigInterface fields from ScopeConfigInterface. 
For example base_url not present in core_config_data and defines in app/code/Magento/Store/Model/Store.php:607.